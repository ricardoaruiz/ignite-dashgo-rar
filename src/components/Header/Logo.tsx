import React from 'react'
import { Text } from '@chakra-ui/react'

export const Logo = () => {
  return (
    <Text fontSize="3xl" fontWeight="bold" letterSpacing="tight" w="64">
      dashgo
      <Text as="span" color="pink.500" fontWeight="bold" fontSize="3xl" ml="1">
        .
      </Text>
    </Text>
  )
}
