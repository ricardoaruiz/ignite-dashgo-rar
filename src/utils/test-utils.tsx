import { render } from '@testing-library/react'
import { ChakraProvider } from '@chakra-ui/react'

import { theme } from '../styles/theme'

export const renderWithTheme = (children: React.ReactNode) => {
  return render(
    <ChakraProvider resetCSS theme={theme}>
      {children}
    </ChakraProvider>
  )
}
