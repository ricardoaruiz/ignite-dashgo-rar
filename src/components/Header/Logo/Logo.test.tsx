import React from 'react'

import { renderWithTheme } from '../../../utils/test-utils'
import { Logo } from './Logo'

describe('<Logo />', () => {
  it('shoud be render correctly by snapshot', () => {
    const { container } = renderWithTheme(<Logo />)

    expect(container.parentElement).toMatchSnapshot()
  })
})
