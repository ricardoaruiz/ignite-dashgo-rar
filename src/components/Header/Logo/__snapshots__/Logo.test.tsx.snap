// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> shoud be render correctly by snapshot 1`] = `
<body
  class="chakra-ui-light"
>
  <div
    id="chakra-toast-portal"
  >
    <ul
      id="chakra-toast-manager-top"
      style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column; margin: 0px auto;"
    />
    <ul
      id="chakra-toast-manager-top-left"
      style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;"
    />
    <ul
      id="chakra-toast-manager-top-right"
      style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;"
    />
    <ul
      id="chakra-toast-manager-bottom-left"
      style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;"
    />
    <ul
      id="chakra-toast-manager-bottom"
      style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column; margin: 0px auto;"
    />
    <ul
      id="chakra-toast-manager-bottom-right"
      style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;"
    />
  </div>
  <div>
    <p
      class="chakra-text css-1y2sdv1"
    >
      dashgo
      <span
        class="chakra-text css-134oou0"
      >
        .
      </span>
    </p>
  </div>
</body>
`
